packages/EasyEntityChange/composer.json

```diff
@@ -4,11 +4,21 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.1",
-        "eonx-com/externals": "^1.0"
+        "php": "^7.3",
+        "eonx-com/standards": "^0.3.0",
+        "phpmd/phpmd": "^2.8",
```

Comment:
Why are those dependencies in `require` and not `require-dev`?

Reply:
Reason: Derping